pkg/daemon/winsvc/winsvc.go

```diff
@@ -18,6 +18,8 @@ type Service interface {
 	Control(svc.Cmd) (svc.Status, error)
 	Query() (svc.Status, error)
 	UpdateConfig(mgr.Config) error
+	RecoveryActions() ([]mgr.RecoveryAction, error)
+	SetRecoveryActions([]mgr.RecoveryAction, uint32) error
```

Comment:
I am not following  this change. Where is the implementation?

Reply:
https://github.com/golang/sys/blob/master/windows/svc/mgr/recovery.go#L63